### DISCUSSION TITLE
Open context of ear trumpet on top instead of bottom

### DISCUSSION THREAD

**Discussion by Cubody** [File-New-Project/EarTrumpet#1614] [Completed ideas]
I have win11 with taskbar on top (by windhawk), is it possible to move context of EarTrumpet on top instead of bottom? Or can it be added please?
![image](https://github.com/user-attachments/assets/3bed6325-92e3-4d90-8d49-e6c1df7ba26c)

**Comment by riverar** [comment]
We support opening on all sides of the display, but looks like you're using some third party software that isn't cooperating. Is this StartAllBack or?

PS: Love the wallpaper. Outer Wilds is one of the best games I've ever played.

**Reply by riverar** [reply]
Oh it's Windhawk, I'll file a bug over there.

**Reply by riverar** [reply]
Looks like this is fixed in Windhawk, https://github.com/ramensoftware/windhawk-mods/issues/1087

**Comment by zORg-alex** [comment]
Windows 11 + Stardock Start11 Taskbar on top flyout opens on bottom.
![image](https://github.com/user-attachments/assets/0b341145-c5f1-4253-b40b-15be03d42720)
Sorry, I've got default wallpaper. Just installed Windows 11 and almost smashed my head over taskbar forced to be on bottom. I'm on ASUS Duo Pro, so it's a no brainer to move it to top, since it's just in the middle of 2 displays.

**Reply by riverar** [reply]
This is a common third-party bug--not a EarTrumpet bug--and is likely related to incorrectly handling of `ABM_GETTASKBARPOS` in Start11. I've passed the feedback on to Stardock.

Another example: https://github.com/ramensoftware/windhawk-mods/issues/1087

**Reply by riverar** [reply]
@zORg-alex Are you on Start11 2.53?

**Reply by riverar** [reply]
@zORg-alex Can you also post at https://forums.stardock.com/536824/start11-253-release-feedback and I'll be sure to ping Sean/Brad. Thanks!

**Reply by zORg-alex** [reply]
I'll try doing it tomorrow

**Reply by NBanfield** [reply]
It looks like whats happened here is someone at Microsoft may have removed all references to the ABM_GETTASKBARPOS message doing anything to the uEdge value.  It now documents that as purely returning the rectangle using the rc member.  Likewise the documentation for the structure lists it as only being used by a few messages and it isn't one of those either.

I guess on Windows 11 the taskbar is on the bottom so someone has decided that useful info is redundant now.

Will be fixed in the next update as we already handled returning that rectangle correctly.

**Reply by riverar** [reply]
@NBanfield Thanks for the quick response Neil! ✊
